submodules/wizard/wizard.js

```diff
@@ -108,11 +108,8 @@ define(function(require) {
 				i18n = self.i18n.active().accountsApp.wizard,
 				i18nSteps = i18n.steps,
 				defaultLanguage = _.get(monster, 'config.whitelabel.language', monster.defaultLanguage),
-				isoFormattedDefaultLanguage = defaultLanguage.substr(0, 3).concat(defaultLanguage.substr(defaultLanguage.length - 2, 2).toUpperCase());
-
-			monster.pub('common.navigationWizard.render', {
-				thisArg: self,
-				data: {
+				isoFormattedDefaultLanguage = defaultLanguage.substr(0, 3).concat(defaultLanguage.substr(defaultLanguage.length - 2, 2).toUpperCase()),
```

Comment:
You should avoid using `string#substr` [has it is loosely depreciated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) (use `string#substring` instead).

Also, `monster.defaultLanguage` is now matched against a [map of valid ISO languages](https://github.com/2600hz/monster-ui/blob/master/src/js/lib/monster.js#L21) so this string formatting should not be necessary anymore.

Reply:
Thanks for the heads up on `string#substr`.

About `monster.defaultLanguage`, I wasn't sure why was that formatting needed, but I left it anyways, just in case :sweat_smile: I'll remove it.